src/univariate/locationscale.jl

```diff
@@ -1,129 +1,131 @@
-LocationScale(μ,σ,ρ) # location-scale transformed distribution
-params(d)            # Get the parameters, i.e. (μ, σ, and the base distribution ρ)
-location(d)          # Get the location parameter
-scale(d)             # Get the scale parameter
+d = σ * ρ + μ       # Create location-scale transformed distribution
+params(d)           # Get the parameters, i.e. (μ, σ, ρ)
 ```
-
-External links
-[Location-Scale family on Wikipedia](https://en.wikipedia.org/wiki/Location%E2%80%93scale_family)
 """
-struct LocationScale{T<:Real, S<:ValueSupport, D<:UnivariateDistribution{S}} <: UnivariateDistribution{S}
+struct AffineDistribution{T<:Real, S<:ValueSupport, D<:UnivariateDistribution{S}} <: UnivariateDistribution{S}
     μ::T
     σ::T
     ρ::D
-    function LocationScale{T,S,D}(μ::T, σ::T, ρ::D; check_args=true) where {T<:Real, S<:ValueSupport, D<:UnivariateDistribution{S}}
-        check_args && @check_args(LocationScale, σ > zero(σ))
+    function AffineDistribution{T,S,D}(μ::T, σ::T, ρ::D; check_args=true) where {T<:Real, S<:ValueSupport, D<:UnivariateDistribution{S}}
+        check_args && @check_args(AffineDistribution, σ > zero(σ))
         new{T, S, D}(μ, σ, ρ)
     end
 end
 
-function LocationScale(μ::T, σ::T, ρ::UnivariateDistribution; check_args=true) where {T<:Real}
+function AffineDistribution(μ::T, σ::T, ρ::UnivariateDistribution; check_args=true) where {T<:Real}
     _T = promote_type(eltype(ρ), T)
     D = typeof(ρ)
     S = value_support(D)
-    return LocationScale{_T,S,D}(_T(μ), _T(σ), ρ; check_args=check_args)
+    return AffineDistribution{_T,S,D}(_T(μ), _T(σ), ρ; check_args=check_args)
 end
 
-LocationScale(μ::Real, σ::Real, ρ::UnivariateDistribution) = LocationScale(promote(μ, σ)..., ρ)
+AffineDistribution(μ::Real, σ::Real, ρ::UnivariateDistribution) = AffineDistribution(promote(μ, σ)..., ρ)
 
 # aliases
-const ContinuousLocationScale{T<:Real,D<:ContinuousUnivariateDistribution} = LocationScale{T,Continuous,D}
-const DiscreteLocationScale{T<:Real,D<:DiscreteUnivariateDistribution} = LocationScale{T,Discrete,D}
+const LocationScale = AffineDistribution
+@deprecate LocationScale(args...; kwargs...) AffineDistribution(args...; kwargs...)
+const ContinuousAffineDistribution{T<:Real,D<:ContinuousUnivariateDistribution} = AffineDistribution{T,Continuous,D}
+const DiscreteAffineDistribution{T<:Real,D<:DiscreteUnivariateDistribution} = AffineDistribution{T,Discrete,D}
 
-Base.eltype(::Type{<:LocationScale{T}}) where T = T
+Base.eltype(::Type{<:AffineDistribution{T}}) where T = T
 
-minimum(d::LocationScale) = d.μ + d.σ * minimum(d.ρ)
-maximum(d::LocationScale) = d.μ + d.σ * maximum(d.ρ)
-support(d::LocationScale) = locationscale_support(d.μ, d.σ, support(d.ρ))
-function locationscale_support(μ::Real, σ::Real, support::RealInterval)
+minimum(d::AffineDistribution) = d.μ + d.σ * minimum(d.ρ)
+maximum(d::AffineDistribution) = d.μ + d.σ * maximum(d.ρ)
+support(d::AffineDistribution) = AffineDistribution_support(d.μ, d.σ, support(d.ρ))
```

Comment:
Can we follow the Julia convention that upper case names are used for types, constructors, and modules only?
```suggestion
support(d::AffineDistribution) = affinedistribution_support(d.μ, d.σ, support(d.ρ))
```

Reply:
Yes! Sorry about that, I should have used case-sensitive search when I was replacing the names of everything.